incubator/tensorflow-inception/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for tensorflow-inception.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+ServiceType: LoadBalancer
+ServicePort: 9090
+ContainerPort: 9090
+Component: "tensorflow-inception"
+Replicas: 1
+Image: "quay.io/lachie83/inception_serving"
+ImageTag: "latest"
```

Comment:
I totally agree. I initially wanted to use the published containers in the original documentation but they don't work currently. Let me work with @viglesiasce to see if we can 't find a better home for this. I will most probably tag it v3 as that's the inception model it's serving.